addons/binding/org.openhab.binding.samsungtv/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0 http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:samsungtv:tv">
+		<parameter name="hostName" type="text" required="true">
+			<label>Host name</label>
+			<description>Network address of the Samsung TV.</description>
+			<context>network-address</context>
+		</parameter>
+		<parameter name="port" type="integer" required="true" min="1" max="65335">
```

Comment:
I didn't test this, but I think required="true" option now change the behavior so that port should always be given by the configuration? Earlier it was optional and if not given then default value was used.

Reply:
I think you are right it should not  be required because there is a default. @cweitkamp can you test and validate?

Reply:
As far as I can see the default value is used independently of the required option. It is possible to create the thing manually in a text file only specifying the hostName parameter like this:
```
Thing samsungtv:tv:livingroom [ hostName="192.168.1.10" ]
```
Discovery or Add a thing unsing Paper UI both preassign the default value for the port. The only difference I observed was that you cannot save the thing configuration in Paper UI if you clear the input field for the port.

Reply:
That is different from what I assumed :-) @kaikreuzer I thought I saw you giving a similar review comment in the past: that default values and required in a mix do not make sense. But the test / explanation of @cweitkamp makes sense as well.

Reply:
Required means that a value has to be given in the config (and thus is mandatory for textually defined things as well). Usually, you do not want to bother users with providing this if there is a default that works for most. So in almost all cases where a default value is given, required should be set to false.

Reply:
I am fine with that. I removed the required option for the port parameter.